core/snippets/snippets/genericForLoopStatement.snippet

```diff
@@ -0,0 +1,10 @@
+name: genericForLoopStatement
+phrase: for
+insertionScope: statement
+---
+
+language: stata
+-
+forval $1 {
```

Comment:
Looks like forval is a ranged loop. This should probably just be moved to the other for snippet